lib/rucio/transfertool/fts3.py

```diff
@@ -354,7 +371,8 @@ def build_job_params(
                       'issuer': 'rucio',
                       'multi_sources': False,
                   },
-                  'overwrite': last_hop.rws.attributes.get('overwrite', overwrite),
+                  'overwrite': overwrite,
```

Comment:
Hello Hugo,
This current implementation LGTM.
 Do you think it would be better to unify/clean how necessary and optional attributes are set in the FTS job config? 

For e.g. `overwrite` and `overwrite_when_only_on_disk` are optional (i.e its okay to not set them, which would suggest a false value in the job config).
In which case we set them how `multihop` is set. 

Also, do you think its useful to set `overwrite_when_only_on_disk` in the `job_metadata`? (One reason behind this is that you can easily see if the flag is set in the job UI). 
I am not sure what transfer groupings are based on and if they need to be considered.

Reply:
Hi Rahul,

I've added the flag to job metadata so is visible as you suggested, I think this is very useful.
The CI is passing now (it was related to the overwrite_corrupted_files setting that I was not aware of).
I've left an image in this PR description explaining which are the the overwrite flags and how they can be tuned.
Regarding group transfers, I haven't seen any showstopper, I think we can go ahead and merge this one. If there are improvements to be made beyond this PR, we can tackle them later.

Reply:
Yes, we need both the config to be set and the `rws` dictionary to be updated with `overwrite=True`  for a resubmit.

Thanks a lot.